default.yml

```diff
@@ -2,108 +2,100 @@
 # https://rubocop.readthedocs.io
 
 AllCops:
-  DisplayCopNames: true
   Exclude:
     - '*.gemspec'
     - 'bin/**/*'
     - 'data/**/*'
-    - 'db/schema.rb'
-    - 'db/data_schema.rb'
+    - 'db/**/*'
     - 'gemfiles/**/*'
-    - 'lib/tasks/circle_0.rake' # Ignore circleci code
+    - 'lib/tasks/circle_0.rake'
     - 'log/**/*'
+    - 'node_modules/**/*'
     - 'public/**/*'
     - 'tmp/**/*'
     - 'vendor/**/*'
+    - 'Guardfile'
 
-Layout/DotPosition:
-  Enabled: false
+Layout/AlignHash:
+  EnforcedLastArgumentHashStyle: ignore_implicit
 
-Layout/ExtraSpacing:
+Layout/AlignParameters:
   Enabled: false
 
-Layout/IndentHash:
-  Enabled: false
+Lint/AmbiguousBlockAssociation:
+  Exclude:
+    - 'spec/**/*'
 
 Layout/IndentArray:
   EnforcedStyle: consistent
 
-Layout/MultilineMethodCallIndentation:
+Layout/CaseIndentation:
   Enabled: false
 
-Layout/SpaceBeforeBlockBraces:
+Layout/ElseAlignment:
   Enabled: false
 
-Layout/SpaceBeforeFirstArg:
+Layout/EndAlignment:
   Enabled: false
 
-Lint/AmbiguousBlockAssociation:
+Layout/IndentationWidth:
   Enabled: false
 
+Layout/IndentHash:
+  EnforcedStyle: consistent
+
+Layout/MultilineMethodCallIndentation:
+  EnforcedStyle: indented
+
 Metrics/AbcSize:
-  Enabled: false
+  Max: 18
 
 Metrics/BlockLength:
   Enabled: false
 
 Metrics/ClassLength:
-  Enabled: false
+  Max: 150
 
 Metrics/CyclomaticComplexity:
-  Enabled: false
+  Max: 7
 
 Metrics/LineLength:
-  Enabled: false
+  IgnoredPatterns: ['(\A|\s)#']
+  Max: 100
 
 Metrics/MethodLength:
-  Enabled: false
-
-Metrics/ModuleLength:
-  Enabled: false
+  Max: 12
 
 Metrics/ParameterLists:
   Enabled: false
 
 Metrics/PerceivedComplexity:
-  Enabled: false
-
-Style/Alias:
-  Enabled: false
+  Max: 8
 
 Style/AndOr:
   EnforcedStyle: conditionals
 
-Style/BracesAroundHashParameters:
+Style/BlockDelimiters:
   Enabled: false
 
-Style/ClassVars:
+Style/BracesAroundHashParameters:
   Enabled: false
 
-Style/CommentAnnotation:
+Style/ClassAndModuleChildren:
   Enabled: false
 
 Style/Documentation:
   Enabled: false
 
-Style/FormatString:
-  Enabled: false
-
 Style/FrozenStringLiteralComment:
   Enabled: false
 
-Style/GuardClause:
-  Enabled: false
-
-Style/Lambda:
+Style/Not:
```

Comment:
these logical operators like `not`, `or` and `and` are a pretty big source of confusion, I think I would rather keep this enabled